jobserver/authorization/roles.py

```diff
@@ -22,18 +22,22 @@
 
 class CoreDeveloper:
     """
-    Bennett staff member with administrator access to Job Server.
+    Temporary vestigial CoreDeveloper role to allow migration
+    """
+
+    display_name = "Legacy CoreDeveloper"
+    description = "Legacy role to be deleted once migration complete"
+    models = []
+    permissions = []
 
-    Note the name is misleading here as this does not imply what we generally mean by
-    "core developer". We plan to rename this role as part of a more general permissions
-    revamp.
+
+class StaffAreaAdministrator:
+    """
+    Bennett staff member with administrator access to Job Server.
```

Comment:
Worth summarising what that means? Access to all of the /staff app, can twiddle most models? Not obvious I think.

Could mention we might split this up in future?

Reply:
The proper description of these roles is decided and agreed on elsewhere, I'd rather not change them in the course of this PR as not to open that particular can of worms.

Reply:
Reasonable, gotcha. Can it say "for info on what that means etc. refer to [XXX]" with a pointer to that elsewhere? Okay if not.

Reply:
Thomas already has a PR in flight to remove these docstrings entirely so I'm going to resolve this because it will become irrelevant after a rebase!